src/ui/Controllers/CourseController.cs

```diff
@@ -40,6 +40,18 @@ public async Task<IActionResult> Variants(string instCode, string accreditingPro
 
             var viewModel = LoadViewModel(org, course, multipleOrganisations);
 
+            if (viewModel.Course.Status.Equals("Running", StringComparison.InvariantCultureIgnoreCase))
```

Comment:
We need to add a test as this should of broken an existing test if it covered it well enough

Reply:
The test was failing and I fixed it